lib/reporter.js

```diff
@@ -86,16 +86,43 @@ module.exports = class Reporter {
     const github = await this.getGithub();
     const login = user.login.toLowerCase();
     const org = this.installation.account.login;
+
     // review:none returns all PRs without a review
     // sadly there is a bug in github that if you commit after
     // someone requested changes, the pr is still reviewed
     // so it will no longer show up whenever someone already reviewed it ever
-    const query = `is:pr is:open review:none review-requested:${login} org:${org}`;
-    const request = github.search.issues({ q: query, per_page: 100 });
-    const pullRequests = await github.paginate(request, response => response.data.items
-      .filter(pr => this.isStale(pr)));
-
-    return _.sortBy(pullRequests, pr => pr.created_at);
+    const toReview = await github.paginate(
+      github.search.issues({
```

Comment:
Refactor out a common `searchIssues` method that attaches `org` and the `per_page` parameter.